### PR TITLE
cleans up contributor guide

### PR DESCRIPTION
**Pull request by agalway-r7**
Reworded the `Contributor.MD` file to hopefully be a little clearer to read for beginners and non-native english speakers.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,27 +40,23 @@ it into Metasploit's master branch.  If you do not care to follow these rules, y
...
-#### New Modules
-
+#### <u>New Modules</u>
+* **Do** license your code as BSD 3-clause, BSD 2-clause, or MIT.
+* **Do** stick to the [Ruby style guide] and use [Rubocop] to find common style issues.
 * **Do** set up `msftidy` to fix any errors or warnings that come up as a [pre-commit hook].
 * **Do** use the many module mixin [API]s.
 * **Don't** include more than one module per pull request.
 * **Do** include instructions on how to setup the vulnerable environment or software.
 * **Do** include [Module Documentation] showing sample run-throughs.
```
Comment (tperry-r7) [inline CONTRIBUTING.md]:
```suggestion
* **Do** include [Module Documentation] showing sample run-throughs.
* **Don't** submit new [scripts].  Scripts are shipped as examples for automating local tasks, and anything "serious" can be done with post modules and local exploits.
* * **Don't** include more than one module per pull request.
```

Comment (tperry-r7) [reply CONTRIBUTING.md]: Move the do and don't to separate sections

**Review by tperry-r7 — COMMENTED** [review]
Made some suggestions. This is great!
@adamgalway-r7

**Comment by adamgalway-r7** [comment]
I added all the above suggestions, but in one commit for the sake of neatness

**Pull request merged**